src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.block {
+  width: 200px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
+
+.photo {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.discription {
+  margin: 36px 16px 0;
+  font-size: 12px;
+  font-weight: 500;
+  width: 166px;
+  height: 36px;
+  align-items: center;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.code {
+  font-size: 10px;
+  margin: 4px 0 0 16px;
+  color: #616070;
```

Comment:
Use CSS variables for repeatable colors.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)